src/main/java/com/esri/core/geometry/WkbExportFlags.java

```diff
@@ -37,5 +37,7 @@
 	public static final int wkbExportMultiPolygon = 32;
 	public static final int wkbExportStripZs = 64;
 	public static final int wkbExportStripMs = 128;
+	public static final int wkbExportAsExtendedWkb = 256;
```

Comment:
I think wkbExportAsExtendedWkb should be removed

Reply:
used here:
https://github.com/davidraleigh/geometry-api-java/blob/ewkb/src/main/java/com/esri/core/geometry/OperatorExportToWkbLocal.java#L102
here:
https://github.com/davidraleigh/geometry-api-java/blob/ewkb/src/main/java/com/esri/core/geometry/OperatorExportToWkbLocal.java#L342
and other spots in the file

Reply:
It seems to be an internal flag though. Unless I am missing something. If it is an internal flag, it should be removed from a public interface. Maybe where it is used, you could pass a boolean instead.

Reply:
https://github.com/Esri/geometry-api-java/commit/b0715e090d6c2b2bc20dff4636549a5ebc6e3ccc#diff-50a693fe943d8766d4616bba6ac739b38613a585dd3463155726c406e6100326R40